changelog/@unreleased/pr-4802.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: fix
+fix:
+  description: |
+    `leaderPaxos` is filtered out from the set of persisted namespaces.
```

Comment:
This is fine, but I would be a bit more specific on explaining the subset of users this actually applied to (i.e. previously *if, and only if, you had used SQLite*, leaderPaxos would be returned as part of the set of persisted namespaces). Otherwise it sounds like it was always being returned

Reply:
👍